src/App.js

```diff
@@ -14,8 +14,9 @@ function App() {
       <Navbar />
       <div
         className={
-          !sidebarHiddenPaths.includes(pathname) &&
-          "d-flex main-content-page-wrapper"
+          !sidebarHiddenPaths.includes(pathname)
+            ? "d-flex main-content-page-wrapper"
+            : undefined
```

Comment:
you can give empty string "" instead of using `undefined`.

Reply:
Ohh yes. I forgot to change this. The better choice here is `null`. I found one [artical](https://www.30secondsofcode.org/articles/s/react-conditional-classname) about this